FreeOTP/RTLSupport.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  RTLSupport.swift
+//  FreeOTP
+//
+//  Created by Vinícius Soares on 09/06/20.
+//  Copyright © 2020 Fedora Project. All rights reserved.
+//
+
+import UIKit
+import TinyConstraints
+
+extension UIView {
+    private var usingDefaultLayoutDirection: Bool {
+        return Self.userInterfaceLayoutDirection(for: self.semanticContentAttribute) != .rightToLeft
+    }
+
+    @discardableResult
+    func rtlLeftToSuperview(offset: CGFloat = 0) -> Constraint {
+        usingDefaultLayoutDirection ? leftToSuperview(offset: offset) : rightToSuperview(offset: -offset)
+    }
+
+    @discardableResult
+    func rtlRightToSuperview(offset: CGFloat = 0) -> Constraint {
+        usingDefaultLayoutDirection ? rightToSuperview(offset: offset) : leftToSuperview(offset: -offset)
+    }
+
+    @discardableResult
+    func rtlLeftToRight(of constrainable: Constrainable, offset: CGFloat = 0) -> Constraint {
+        usingDefaultLayoutDirection ? leftToRight(of: constrainable, offset: offset) : rightToLeft(of: constrainable, offset: -offset)
+    }
+
+    @discardableResult
+    func rtlRightToLeft(of constrainable: Constrainable, offset: CGFloat = 0) -> Constraint {
+        usingDefaultLayoutDirection ? rightToLeft(of: constrainable, offset: offset) : leftToRight(of: constrainable, offset: -offset)
+    }
+}
```

Comment:
This code can be moved into `TokensViewController.swift`, I don't think it needs to be in its own individual file.

Reply:
or `TokenCell.swift`

Reply:
I think this may be used elsewhere if other views start to be made using view code (Which seems a trend in the community).

Note that this extension is only required because the app still supports iOS 9.